fineract-provider/src/main/java/org/apache/fineract/cob/loan/ApplyChargeToOverdueLoansBusinessStep.java

```diff
@@ -28,6 +29,7 @@
 
 @Component
 @RequiredArgsConstructor
+@BulkEventSupport
```

Comment:
Can we verify that this setting is not conflicting global configuration setting for bulk events for Loan cob.

Reply:
The aspect calls the already existing `BusinessEventNotifierService` methods that enable or disable the bulk logging flag. The flag is still checked in the `notifyPostBusinessEvent` method after the global flag check.